src/tools/rustdoc-gui/tester.js

```diff
@@ -70,6 +70,14 @@ function parseOptions(args) {
     return null;
 }
 
+function print_test_successful() {
+    process.stdout.write(".");
+}
+function print_test_erroneous() {
+    // Bold Red "F" Reset
+    process.stdout.write("\033[1m\x1b[31mF\x1b[0m");
```

Comment:
That might work well for linux, but not on windows at all. ;)

Reply:
I wished that JavsScript had `#[cfg(unix)]` ...

Reply:
If that's the only thing you wished js had, I guess it's fine. 😉 

For now, just keep it simple and simply print text.

Reply:
I found an easy way to detect linux at runtime and enable the colour, what do you think?

Reply:
I
 think it's completely beyond scope. I'd rather keep it as simple as possible. Please simply print the text (on stderr if it's an error).

Reply:
Done :+1: